cl_dll/ammohistory.cpp

```diff
@@ -138,7 +138,9 @@ int HistoryResource :: DrawAmmoHistory( float flTime )
 				}
 
 				// Draw the number
-				gHUD.DrawHudNumberString( xpos - 10, ypos, xpos - 100, rgAmmoHistory[i].iCount, r, g, b );
+				char szString[32];
+				sprintf( szString, "%d", rgAmmoHistory[i].iCount );
```

Comment:
Just in case could you use `snprintf` here please?

Reply:
For sure, it was a direct copy paste from DrawHudNumberString. Done.

Reply:
@YaLTeR I've noticed that the `DrawHudNumberString` function is no longer called by anything as it was only called in this instance, should I change it there instead?

Reply:
Yeah, that would make sense I guess.

Reply:
Never mind, it's used in `scoreboard.cpp` which isn't used _right now_, but could be if one were to implement this: https://github.com/YaLTeR/OpenAG/issues/89 so I'd say for compatibility, let's keep the DrawHudNumberString unchanged. Sorry about that.

Reply:
So this PR is good to go now right?

Reply:
Yep.